### DISCUSSION TITLE
还是请教连接binance的问题，vnpy程序定期跑一段时间就会卡死

### DISCUSSION THREAD

**Discussion by speedfengjun** [veighna-global/vnpy_evo#22] [General]
ddddddd

**Comment by xldistance** [comment]
实盘需要使用no_ui运行，至于接口连接重连可以在rest api和websocket api里面的except里面维护交易接口的连接状态，比如{self.gateway_name:True},no_ui的父进程里面定时读取这个状态如果是False(因为异常导致连接断开)就重启交易子进程交易然后再赋值状态为True

**Comment by ttttheman** [comment]
是的，过一段时间就收不到on_tick了。有人碰到类似的问题吗，如何解决的？
